src/components/notifications/SMSPartner.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="mb-3">
+        <label for="smspartner-key" class="form-label">{{ $t("API Key") }}</label>
+        <HiddenInput id="smspartner-key" v-model="$parent.notification.smspartnerApikey" :required="true" autocomplete="new-password"></HiddenInput>
+        <div class="form-text">
+            <i18n-t keypath="smspartnerApiurl" as="div" class="form-text">
+                <a href="https://my.smspartner.fr/dashboard/api" target="_blank">https://my.smspartner.fr/dashboard/api</a>
```

Comment:
bit personal preference, but feel like this can be shortened ^^

```suggestion
                <a href="https://my.smspartner.fr/dashboard/api" target="_blank">my.smspartner.fr/dashboard/api</a>
```